packages/sdk/src/rtl/browserTransport.ts

```diff
@@ -100,6 +79,93 @@ export class BrowserTransport extends BaseTransport {
     super(options)
   }
 
+  /** Does this browser have the necessary performance APIs? */
+  static supportsPerformance() {
+    return (
+      performance &&
+      performance.mark !== undefined &&
+      performance.measure !== undefined
+    )
```

Comment:
performance && !!performance.mark && !!performance.measure

Reply:
ok. Mozilla examples showed them explicitly checked, but maybe that was for javascript?